src/components/HoldMenuSectionList.js

```diff
@@ -0,0 +1,59 @@
+import _ from 'lodash';
+import React from 'react';
+import {View} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@styles/useThemeStyles';
+import variables from '@styles/variables';
+import Icon from './Icon';
+import * as Illustrations from './Icon/Illustrations';
+
+function HoldMenuSectionList() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const holdMenuSections = [
+        {
+            icon: Illustrations.Hourglass,
+            titleTranslationKey: 'iou.whatIsHoldTitle',
+            descriptionTranslationKey: 'iou.whatIsHoldExplain',
+        },
+        {
+            icon: Illustrations.CommentBubbles,
+            titleTranslationKey: 'iou.holdIsTemporaryTitle',
+            descriptionTranslationKey: 'iou.holdIsTemporaryExplain',
+        },
+        {
+            icon: Illustrations.TrashCan,
+            titleTranslationKey: 'iou.deleteHoldTitle',
+            descriptionTranslationKey: 'iou.deleteHoldExplain',
+        },
+    ];
```

Comment:
Wouldn't it make more sense to pass these as props to make this component reusable? Although we may not use it in the future it would be cleaner IMO

Reply:
I'm not sure about this one, I separated it for now as you suggested but to avoid code repetition I also had to pass that value as a prop to both Menu and Page, which might not be what we want. Let me know what you think.